assets/src/application/ui/layout/entityList/filterBar/buttons/FilterButtonWrap.tsx

```diff
@@ -5,7 +5,7 @@ import { FilterButtonWrapProps } from '../types';
 
 const FilterButtonWrap: React.FC<FilterButtonWrapProps> = ({ children, id, label }) => {
 	return (
-		<div className='ee-filter-bar-filter-btn-wrap'>
+		<div className='ee-entity-list-filter-bar__btn__wrap'>
```

Comment:
I am not a BEM master, hope this is ok :)

Reply:
ya I'm certainly not a master either considering I just discovered this recently but I don't think I've ever seen nested elements like that. I think that would go against at least part of what BEM is trying to achieve.

it appears the above violation happens because you are merging two block names together

ie: entity list is one block and filter bar is another block vs the filter bar being an element of the entity list. 

yes you _could_ look at the filter bar being an element of the entity list, but it's probably far more accurate to view it as its own thing and therefore as a separate block.

So that said, if you remove `entity-list` from that selector, then it's easier to fit into the BEM schema:

- BLOCK: `ee-filter-bar`
- ELEMENT: `btn-wrap`
- MODIFIER: ???

with the resulting selector being: `ee-filter-bar__btn-wrap`

Then for the Entity List block the elements might be things like:

- `ee-entity-list__header` <== insert filter bar here
- `ee-entity-list__body` <== the actual entity list goes here (or maybe `ee-entity-list__list`)
- `ee-entity-list__footer` <== pagination and other buttons go here

A possible good rule of thumb for differentiating something as belonging to its own block might be to look if it is in its own folder or not. So in this case, despite the filter bar being within the `entityList` folder, it is also within its own `filterBar` folder and therefore should be considered its own block. I'm just making up that rule on the spot, it's not anything I have read anywhere so I am (as always) totally open for suggestions, etc.

Reply:
Improved that button wrap: https://github.com/eventespresso/event-espresso-core/pull/2578/commits/ca40c5f1c1823b1dabfbcb818385d48b128ffb4c